openapi.yaml

```diff
@@ -22735,28 +22735,20 @@ components:
           x-linode-filterable: true
           type: string
           readOnly: true
-          description: >
-            The class of the Linode Type. We currently offer five classes of Linodes:
-
-              * nanode - Nanode instances are good for low-duty workloads,
-                where performance isn't critical. **Note:** As of June 16th, 2020, Nanodes became
-                1 GB Linodes in the Cloud Manager, however, the API, the CLI, and billing will
-                continue to refer to these instances as Nanodes.
-              * standard - Standard Shared instances are good for medium-duty workloads and
-                are a good mix of performance, resources, and price. **Note:** As of June 16th, 2020,
-                Standard Linodes in the Cloud Manager became Shared Linodes, however, the API, the CLI, and
-                billing will continue to refer to these instances as Standard Linodes.
-              * dedicated - Dedicated CPU instances are good for full-duty workloads
-                where consistent performance is important.
-              * gpu - Linodes with dedicated NVIDIA Quadro &reg; RTX 6000 GPUs accelerate highly
-                specialized applications such as machine learning, AI, and video transcoding.
-              * highmem - High Memory instances favor RAM over other resources, and can be
-                good for memory hungry use cases like caching and in-memory databases.
-                All High Memory plans contain dedicated CPU cores.
+          description: |
+            The class of the Linode Type. We currently offer six classes of compute instances:
```

Comment:
Can we add a general note to this section that some of these types are only available in some regions, and that folks can check the regions capabilities list to determine where these are supported?

Reply:
yes, great suggestion